### DISCUSSION TITLE
How to use existing clangd binary

### DISCUSSION THREAD

**Discussion by juxeii** [mason-org/mason.nvim#1356] [Q&A] [answered]
Hi,

I have an environment, where I cannot use the latest `clangd` LSP for C++.
How can I tell `mason.nvim` to look for an existing binary on the host, e.g. `which clangd`, and use this as the default. 
If no binary is found, `mason.nvim` should install the latest `clangd`.
Is this possible?

**Comment by williamboman** [comment, marked as answer]
Mason doesn't "use" binaries in any way, it only installs them and makes them available in Neovim via PATH. Don't install clangd via Mason if you want to use a different installation.

**Comment by juxeii** [comment]
Ok, thx.
